test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -570,11 +570,26 @@ Describe "Hard link and symbolic link tests" -Tags "CI", "RequireAdminOnWindows"
             $ci[1].Name | Should -MatchExactly $filenamePattern
             $ci[2].Name | Should -MatchExactly $filenamePattern
         }
+        It "Get-ChildItem -Name gets content of linked-to directory" {
+            # The test depends on the files created in previous test:
+            #$filenamePattern = "AlphaFile[12]\.txt"
+            #New-Item -ItemType SymbolicLink -Path $alphaLink -Value $alphaDir
+            $ci = Get-ChildItem $alphaLink -Name
+            $ci.Count | Should -Be 3
+            $ci[1] | Should -MatchExactly $filenamePattern
+            $ci[2] | Should -MatchExactly $filenamePattern
+        }
         It "Get-ChildItem does not recurse into symbolic links not explicitly given on the command line" {
```

Comment:
Should have newline after closing brace and `It`

Reply:
This is current pattern for the file. I don't want reformat all file in the PR.